File: core-java-program/gcr-codebase/programming-elements/PowerCalc.java
import java.util.Scanner;
public class PowerCal{
	public static void main(String[] args){
		Scanner sc = new Scanner(System.in);
		System.out.print("Enter base: ");
		int base = sc.nextInt();
		System.out.print("Enter exponent: ");
		int exponent = sc.nextInt();
		int power = (int)(Math.pow(base, exponent));
		System.out.println("Answer: " + power);
	}
}

File: core-csharp-program/gcr-codebase/programming-elements/PowerCalc.cs
using System;

public class PowerCalc
{
    public static void Main(string[] args)
    {
        Console.Write("Enter base: ");
        int baseNum = int.Parse(Console.ReadLine());

        Console.Write("Enter exponent: ");
        int exponent = int.Parse(Console.ReadLine());

        int power = (int)Math.Pow(baseNum, exponent);
        Console.WriteLine("Answer: " + power);
    }
}
